generators/vsi/templates/terraform/scripts/start_master.sh

```diff
@@ -29,3 +29,8 @@ pkill java
 unzip {{deployment.name}}.zip
 wlp/bin/server start
 {{/has}}
+{{#has deployment.language 'GO'}}
+cd /usr/src/{{deployment.name}}
+pkill go
```

Comment:
From my understanding, GoLang binaries are able to run directly, and do not need/use Go to do so. Also, I don't think your implementation of deploying GoLang on VSI installs Go on the VSI, so I don't think `pkill go` will do anything. You might have to do a little bit of research here to figure out how to kill previous go_executable processes.